pkg/commands/workflows/remove_comments.go

```diff
@@ -71,6 +84,13 @@ func (c *RemovePlanCommentsCommand) Flags() *cli.FlagSet {
 		Usage:   "The GitHub pull request number to remove plan comments from.",
 	})
 
+	f.StringSliceVar(&cli.StringSliceVar{
+		Name:    "comment-type",
+		Target:  &c.flagCommentTypes,
+		Example: "true",
```

Comment:
`true` doesn't seem right here, maybe a copy-paste oversight

Reply:
Gah, good catch, this was from an earlier implementation.